.travis.yml

```diff
@@ -238,6 +240,7 @@ jobs:
         - pip3 install -U pip
         - pip3 install PyYAML
         - pip3 install python-dateutil
+        - pip3 install paramiko
```

Comment:
We don't need paramiko to test the C++ autograding code?

Reply:
Paramiko is now needed to run `INSTALL_SUBMITTY.sh`. SSH is not used, but paramiko is imported into `update_and_install_workers.py`.